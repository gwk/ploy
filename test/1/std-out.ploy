main = outL.'this is std_out.'
