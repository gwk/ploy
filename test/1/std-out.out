this is std_out.
